policyengine_us/variables/gov/states/de/tax/income/subtractions/pension_exclusion/de_pension_exclusion.py

```diff
@@ -0,0 +1,84 @@
+from policyengine_us.model_api import *
+
+
+class de_pension_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware pension exclusion"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf#page=6"
+        "https://delcode.delaware.gov/title30/c011/sc02/index.html"
+    )
+    defined_for = StateCode.DE
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.de.tax.income.subtractions.pension_exclusion
+        person = tax_unit.members
+        # determine age eligibility
+        age_head = tax_unit("age_head", period)
+        under_60_head_eligible = (age_head < p.min_age).astype(int)
+        above_60_head_eligible = (age_head >= p.min_age).astype(int)
+
+        # determine military eligiblity
+        military_retirement = person("military_retired", period) > 0
+        military_eligible = tax_unit.any(military_retirement)
+
+        # determine pension exclusion value based on military status
+        exclusion_value = where(
+            military_eligible,
+            p.military_pension_amount_cap,
+            p.non_military_pension_amount_cap,
+        )
+
+        # determine pension exclusion amount
+        pension_income = person("market_income", period)
+
+        # determine eligible retirement income for head above 60
+        elig_retirement_income = person(
+            "eligible_retirement_income_for_elderly", period
```

Comment:
this variable does not exist we want to use a combination of existing retirement income and age